CHANGELOG.md

```diff
@@ -1,39 +1,41 @@
+## 1.1.0
+
+- Remove unnecessary `clock` dependency and use the built-in `Stopwatch` object
+  instead.
+- Pass an instance of the `PausableTimer` in the callback.
+
 ## 1.0.0+5
 
 - Update and test against Dart 2.17.
 
-[More details available in
-GitHub](https://github.com/llucax/pausable_timer/milestone/9?closed=1).
+[More details available in GitHub](https://github.com/llucax/pausable_timer/milestone/9?closed=1).
```

Comment:
Could you remove all the reformatting changes from the changelog?

Reply:
> Could you remove all the reformatting changes from the changelog?

What is wrong with them? The file was automatically formatted by the Dart formatter when I saved it.

Reply:
That is markdown, not dart, so maybe some other tool did them? There is nothing wrong with them, but there is nothing wrong with how it is formatted either, so I don't see any reason to change it and add noise to the diff. It is distracting and maybe the next person that opens a PR thinks there is a better formatting too (or has another tool to automatically format it). It just makes no sense to change it, unless it is a standard tool that is making the auto formatting and we can add a check to the CI to make sure the same format is always used (as it is with the dart code).